adminPage/adminPageActions.php

```diff
@@ -1,38 +1,25 @@
 <?php
-require '../dbConn.php';
+require_once '../dbConn.php';
+require_once '../functions.php';
 
 $projectsTitle = $_POST['portfolioTitle'];
 $projectsText = $_POST['projectText'];
 $projectsUrl = $_POST['projectUrl'];
 
-$projectsInsertQuery = $db->prepare("INSERT INTO `projects` (`title`, `fillText`, `url`) VALUES (:projectsTitle, :projectsText, :projectsUrl)");
-$projectsInsertQuery->bindParam(':projectsTitle',$projectsTitle);
-$projectsInsertQuery->bindParam(':projectsText',$projectsText);
-$projectsInsertQuery->bindParam(':projectsUrl',$projectsUrl);
-$projectsInsertQuery->execute();
-
-/*
- * Function that is designed to take in the db, select the projects table and fetch all its data. Being used to retrieve the data displayed on the admin page.
- *
- * @param - called $db, designed to take the pre-existing $db variable
- *
- * @return - an array - of all info in the projects table on the db
- */
-function retrieveProjectsData($db) {
-    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
-    $projectsEditQuery = $db->prepare("SELECT * FROM `projects`;");
-    $projectsEditQuery->execute();
-    $projectsResult = $projectsEditQuery->fetchAll();
-    return $projectsResult;
-}
-$projectsArray = retrieveProjectsData($db);
-
 $projectsDeleteId = $_POST['deleteProjectId'];
-$projectsDeleteQuery = $db->prepare("DELETE FROM `projects` WHERE `projects`.`id` = :deleteId;");
-$projectsDeleteQuery->bindParam(':deleteId',$projectsDeleteId);
-$projectsDeleteQuery->execute();
 
-////comments here
+$aboutMeText = $_POST['aboutMeText'];
+$techText = $_POST['techText'];
+$prevExpText = $_POST['prevExpText'];
+$otherExpText = $_POST['otherExpText'];
+
+insertProject($db, $projectsTitle, $projectsText, $projectsUrl);
```

Comment:
try to create logic that only call these functions if they are needed not just all of them by default